packages/core/parcel-bundler/src/Resolver.js

```diff
@@ -405,6 +405,10 @@ class Resolver {
           }
         }
       }
+      // Or try a lookup replacing backslash characters with forward slash
+      if (alias == null && ~filename.indexOf('\\')) {
+        alias = aliases[filename.replace(/\\/g, '/')];
```

Comment:
Should we use `path.normalize` for these operations?

Reply:
Using `path.normalize` will remove `.` and `..` and collapse double slashes. That wouldn't fix the bug we're solving here, which is just converting backslashes to forward.